src/renderer/components/StandardButton.tsx

```diff
@@ -0,0 +1,5 @@
+import { Button } from '@mui/material';
+
+const StandardButton = Button;
+
```

Comment:
Assume this is a place holder to add styling later?

Reply:
Yes. Definitely. Just trying to get everyone using a standardised button and someone can come in and do the props / forwarding, etc.